### ISSUE TITLE
Learning Hub v2: filter content by usage

### ISSUE THREAD

**Issue by acressall** [ParabolInc/parabol#6929]
Update the learning hub experience to be different for users based on their usage of the product. 

- 0 meetings participated in (existing empty meeting state)
- 1 meeting participated
- 2-4 meetings participated
- 4+ meetings participated

**Comment by Dschoordsch** [comment]
I would also like to not distinguish between no meetings and open meetings, but rather put it always put the learning content at the bottom/end with some separation.

**Comment by acressall** [comment]
@Dschoordsch a _potential_ update to the design. WDYT?
https://www.loom.com/share/a12cf1cc74e140da9db7f641782d6395

**Comment by acressall** [comment]
Here's the logic I'm thinking: https://www.notion.so/parabol/Squad-2-Notes-515904adde5441febd81750e4c7a1e65?p=e25be14b44c642cf8d8592c65e3f5465&pm=s

I based it only on meeting participation for now. I'd like to look at clicks on these cards to see how people engage

- Are more meetings run?
- Are more integrations added?
- Are more meetings viewed from the timeline?

If we notice an improvement we can add more criteria, such as integrations added, number of teams, number of teammates, etc

**Comment by Dschoordsch** [comment]
I really like the design. I like the idea of adding a add meeting card where people look for their meetings. Also having the learning cards less popping compared to actual meetings will help distinguish these. I wonder how we would incorporate the current tutorial video visually.

**Comment by Dschoordsch** [comment]
@acressall Once this issue has passed design review, please create an implementation issue in "Todo" for it.

**Comment by acressall** [comment]
@Dschoordsch I would like to propose we tackle this work in phases:

1. Update the copy and styling (to reduce accidental clicks on the current demo card)
![Image](https://user-images.githubusercontent.com/18582225/181094111-91925075-4304-4a2b-8a74-319c4c5fb350.png)

2. [Add logic](https://www.notion.so/parabol/Squad-2-Notes-515904adde5441febd81750e4c7a1e65?p=e25be14b44c642cf8d8592c65e3f5465&pm=s) to which content can be shown to which users (still being refined)
3. Start a meeting btn before the meeting cards can be looked at separately, with the #6720

**Comment by acressall** [comment]
@enriquesanchez can you give the new card style a review? There's rationale as to the new style in the Loom above.
I expect these to behave very similarly to the current meeting cards in terms of layout, simply narrower. 
The video tutorials will open a modal where they can be played (which is what we have now). 

I would especially like your opinion on:

- Images: should we use illustrations, or will product / tutorial faces being more engaging and differentiating from meeting cards?
- Should items like the invite link open a _tutorial_ in a modal, or open the actual invite modal? What would you expect as a user? The tricky part here is it's from the meeting view, so we won't know _which team_ they want to invite to. I've tried to make that clear in the copy, but it might still be confusing
- Same question 👆 for integrations

**Comment by enriquesanchez** [comment]
@acressall I left some comments on the Figma file as well.

>Images: should we use illustrations,

I think illustration work well here, it follows the theme and style we use elsewhere. One thing I suggested on the Figma file was to use distinct illustration from the main ones used for the meetings. So for example, I think the Sprint Poker Scales illo works really well and feels fun, but the one for Invite your team feels a bit too similar to the main ones.

>Should items like the invite link open a tutorial in a modal, or open the actual invite modal? 

🤔 I would expect for it to open the invite modal.... but you are correct on the tricky part here. If I have multiple teams, how would we know which one to invite to?

Seeing this potential confusion, maybe then this should open a tutorial on how to invite folks to a team?

>Same question 👆 for integrations

Yep, since this is a multi-team view, and integrations are a per-team setting, maybe we show the tutorial for how to?

**Comment by acressall** [comment]
@ackernaut this is ready for review.
- Updated logic: https://www.notion.so/parabol/Squad-2-Notes-515904adde5441febd81750e4c7a1e65?p=e25be14b44c642cf8d8592c65e3f5465&pm=s
- Images are placeholders
- Based on feedback above, only the demo will open in a new tab. All other cards will open a modal
- We'll show a min 3, max 4 cards at a time. It should not take more than 1 row on desktop to as not to take away from the primary purpose of the meetings view (these are _supplementary content_ to meetings)

I'd particularly like your thoughts on the new card style... because it's new. Reasons for the new style:

- Distinguish from the meeting cards (they will be visible in both the empty and not-empty meeting view)
- Make them _feel_ like learning (book-ish shaped)

I'm open to hearing pros/cons to adding a new card style

**Comment by ackernaut** [comment]
Thank you, taking a card!

**Comment by ackernaut** [comment]
Hey! I took a look at Loom, Notion, Figma. I have some questions:
- You suggest 3-4 min/max cards, why that many cards?
    - We have 2 cards today, what have we learned so far about engagement?
- If I take an action, does the card go away?
    - If I invite a user to my team, does the invite card go away?
    - If I try the demo from the card?
- Can a user remove the cards if they want?
- Does this content live anywhere else? You have about 14 potential cards listed, anyway to access all of that?
- Are you planning to track clicks on individual cards?
- Are you planning to track engagement with content once the card is open (they watched the video, the clicked the CTA (e.g. Start Sprint Poker) in that dialog)?
- How much content do you have to start this? You mention User Stages but do you have the content to support all of those? Otherwise, what are the first tracks and criteria for showing based on what you do have?
- Should we remove timeline actions?

**Comment by acressall** [comment]
Thanks for the great questions @ackernaut!

> You suggest 3-4 min/max cards, why that many cards?

I'd like to fill a row on desktop, but not overwhelm with help content (and keep the focus on meetings when they exist)

> We have 2 cards today, what have we learned so far about engagement?

I can double check on the video, but the engagement on the demo has been _high_. This has also resulted in higher dropoff, so I suspect some users mistake it for a meeting. That's where the new card style will be a help too

> If I take an action, does the card go away?

No, not at this stage. I think this would be a nice improvement though. We could add it as part of the functionality, but add it in phases

> Can a user remove the cards if they want?

Not at this stage, but I think it would be nice to do as an enhancement at some point

> Does this content live anywhere else? You have about 14 potential cards listed, anyway to access all of that?

Not yet, but I do want to think about another spot where learning content could live, or maybe even community stuff. For now, I want to focus on the initial _I just signed up_ experience and keep it very visible. I could see us combining approaches in future -- learning hub for new users, and a separate permanent place for everyone

> Are you planning to track clicks on individual cards?

Yes, I'd like to test the overall pattern, but also which content users are looking for the most, so we can refine that

> Are you planning to track engagement with content once the card is open (they watched the video, the clicked the CTA (e.g. Start Sprint Poker) in that dialog)?

The way I'm thinking about it is it would be interesting to see if users who engage with any of these cards tend to become more engaged users overall (similar to what we know about integrations). Hypothesis: encouraging small, non-committal actions will make users more likely to continue taking action

> How much content do you have to start this? You mention User Stages but do you have the content to support all of those? Otherwise, what are the first tracks and criteria for showing based on what you do have?

We don't have content yet, my thought was it could happen in parallel with development. I want to make sure we're creating the right things by figuring out filtering users first. Active discussion happening here https://parabol.slack.com/archives/C03GZQJ27MG/p1660050122267849

> Should we remove timeline actions and seed task cards that prompt the same actions?

hmm I don't want to be redundant, but I'm hesitant to remove them just yet. My hesitation is that the seed task cards give context to that view as well, so I like having _something_ there. I'm less tied to the timeline actions, since there is a stronger overlap between their purpose and this

**Comment by ackernaut** [comment]
Digging into the Figma some more, I’ll leave some comments there!

**Comment by acressall** [comment]
@ackernaut designs updated! LMK if you think this is good to go!

**Comment by ackernaut** [comment]
@acressall just [one more question here](https://www.figma.com/file/5qyWAMoB0jZAY195ECZDf7?node-id=138:895#249725298) around the existing onboarding cards

**Comment by ackernaut** [comment]
Nice work!! I think the new onboarding card pattern is a solid improvement to help them feel distinct from meeting cards in the view.
